common.css

```diff
@@ -0,0 +1,14 @@
+body
+{
+    background-color: var(--main-background-color);
+    box-shadow: 0 4px 12px 0 rgb(0 0 0 / 5%);
+    font-family: Whitney,-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica,Arial,sans-serif;
+}
+*
+{
+    padding: 0;
+    margin: 0;
```

Comment:
you can add box-sizing : border-box property here

Reply:
Done